benchmark/clblast/blas1/iamin.cpp

```diff
@@ -50,6 +50,28 @@ void run(benchmark::State& state, ExecutorType* executorPtr, index_t size) {
   MemBuffer<scalar_t, CL_MEM_WRITE_ONLY> buf1(executorPtr, v1.data(), size);
   MemBuffer<int, CL_MEM_READ_ONLY> buf_i(executorPtr, &res, 1);
 
+  // No verification for CLBlast's Amin until it is fixed
```

Comment:
Why there is commented code here?

Reply:
There was a problem upstream, it has been fixed now so I uncommented